docker/alt.docker

```diff
@@ -34,8 +34,11 @@ COPY frameworks/php frameworks/php
 COPY test/runner.js test/
 COPY test/runners/php_spec.js test/runners/
 
+COPY entrypoint.sh entrypoint.sh
+RUN chmod +x entrypoint.sh
+
 USER codewarrior
 ENV USER=codewarrior HOME=/home/codewarrior
 RUN mocha -t 3000 test/runners/php_spec.js
 
-ENTRYPOINT ["node"]
+ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
Why does this have `entrypoint.sh` changes?

Reply:
I'm not sure what happened. I think it was at this point that I starting using origin vs upstream. A rebase might help but I don't want to risk spending the next 30 minutes fixing something if it messes up the history even more.